SwiftUIHW/SwiftUIHW/Login/LoginView.swift

```diff
@@ -0,0 +1,200 @@
+// LoginView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Страница входа
+struct LoginView: View {
+    // MARK: - Public properties
+
+    var body: some View {
+        ZStack {
+            LoginView.gradientBackgroundView
+            ZStack {
+                Color.white.ignoresSafeArea(.all, edges: .bottom)
+
+                VStack(spacing: Constants.firstVStackSpacingValue) {
+                    logInSingUpView
+                    loginAndPasswordTextFieldsView
+
+                    VStack {
+                        getStartedButtonView
+                        forgotPasswordButtonView
+                    }
+                    .padding(.top, Constants.firstVStackSpacingValue)
+                }
+            }
+        }
+        .navigationBarBackButtonHidden(true)
+        .toolbar(content: {
+            ToolbarItem(placement: .navigationBarLeading) {
+                customDismissButtonView
+            }
+        })
+        .onTapGesture {
+            hideKeyoard()
+        }
+    }
+
+    // MARK: - Private properties
+
+    @Environment(\.dismiss) private var dismiss
+    @FocusState private var focusedField: Int?
```

Comment:
```suggestion

    @FocusState private var focusedField: Int?
    
```
Отступы между разными враперами